### PR TITLE
Update formatting, split tests, some spacing

### LINKED ISSUE #80: Improve Slack notification formatting
Currently the messages look a little bland, let's jooze them up by supporting multiline grouping like

```
*Default*
_Action:_ red
_Background:_ pink
*Dark*
_Focus:_ green
```

and maybe convert the values to hex so slack picks it up as a color

### PR DESCRIPTION
**Pull request by alex-page**
This will fix #80

### CODE REVIEW

**Comment by alex-page** [comment]
Write a test for SendChameleonMessage...

File: src/index.js [hunk 1]
```diff
@@ -40,11 +40,13 @@ App.get( `${ Settings.endpoint }*`, async ( request, response ) => {
...
 
-	// Notify Slack! 
-	if ( process.env.VCAP_SERVICES ) {
-		SendSlackMessage( `_Karma-Karma-Karma-Chameleon!_ \nGenerating \`${ParseRequestPath( request.path )}\` template using... ${ColorMapToString( request.query )}` );
+	// Notify Slack!
+	if( process.env.VCAP_SERVICES ) {
+		SendChameleonMessage( request.path, request.query );
 	}
 
+	SendChameleonMessage( request.path, request.query );
```
Comment (adamzerella) [inline src/index.js]: This should error out as we only want to send when `VCAP_SERVICES` is set ?

File: test/unit/slack.spec.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		Expect( GetTemplateFromURL( '/chameleon' ) ).to.equal( 'homepage' );
+		done();
+	});
+
+	It( 'Should return the homepage template when requesting /chameleon/', ( done ) => {
+		Expect( GetTemplateFromURL( '/chameleon/' ) ).to.equal( 'homepage' );
+		done();
+	});
+
+	It( 'Should return the basic template when requesting /chameleon/basic', ( done ) => {
```
Comment (adamzerella) [inline test/unit/slack.spec.js]: Might be safe to also add a test for `/chameleon/basic/`

Comment (adamzerella) [reply test/unit/slack.spec.js]: Added the test case

**Pull request merged**